clients/src/test/java/org/apache/kafka/clients/consumer/internals/FetcherTest.java

```diff
@@ -2035,6 +2036,29 @@ public void testConsumingViaIncrementalFetchRequests() {
         assertEquals(5, records.get(1).offset());
     }
 
+    @Test
+    public void testFetcherWithinBlackoutPeriodAfterAuthenticationFailure() {
```

Comment:
This test case seems like overkill since we're already checking the `beginningOffsets` path in the `KafkaConsumer` tests.

Reply:
Sure, that makes sense. I'll remove this redundancy too.